src/components/MapControls/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react'
+import { NavigationControl, GeolocateControl } from 'react-map-gl'
+
+export const MapControls: FC<{
+  isMobile: boolean
+}> = ({ isMobile }) => {
+  return (
+    <div className={`absolute right-4 ${isMobile ? 'top-4' : 'bottom-4'}`}>
```

Comment:
Similar to my other comment, I would suggest to leave the positioning of the controls to the parent in which they are rendered.